src/modules/TodoStore.js

```diff
@@ -0,0 +1,58 @@
+import { getTodos, setTodos } from './Storage.js';
+
+export default class TodoStore {
+  constructor() {
+    this.getData();
+  }
+
+  getData() {
+    this.todosArray = getTodos();
+  }
+
+  // getNextIndex() {
+  //   return this.todosArray.length + 1;
+  // }
```

Comment:
- [ ] Please remove commented code as they make your code base look cluttered and difficult to read 👍